### PR TITLE
fix #87

### LINKED ISSUE #87: keep 同步脚本，可能是个 bug，待确认
https://github.com/yihong0618/running_page/blob/06d308ebc678b47429a5c7806786b8c028022160/scripts/keep_sync.py#L51

```
i["logs"][0]
```

这里好像只获取每天的最后一条记录，如果一天有多条数据，较早的数据被跳过了。
（只看了代码觉得有问题，还没有实测）

### PR DESCRIPTION
**Pull request by Lax**

### CODE REVIEW

File: scripts/keep_sync.py [hunk 1]
```diff
@@ -48,7 +48,8 @@ def get_to_download_runs_ids(session, headers):
         r = session.get(RUN_DATA_API.format(last_date=last_date), headers=headers)
         if r.ok:
             run_logs = r.json()["data"]["records"]
-            result.extend([i["logs"][0]["stats"]["id"] for i in run_logs])
+            for i in run_logs:
+                result.extend(j["stats"]["id"] for j in i["logs"])
```
Comment (yihong0618) [inline scripts/keep_sync.py:52]: I think its the same logic

Comment (yihong0618) [reply scripts/keep_sync.py:52]: Ha, get your point, maybe there's no log here?

**Comment by yihong0618** [comment]
感谢·

**Comment by yihong0618** [comment]
@Lax 
顺便说一下， keep 目前只拿了 running 的数据，如果有走路爬山自行车的数据需要改type
image
比如骑行是cycling
具体对应的字段可以在这个 api 找到 https://api.gotokeep.com/config/v2/basic?refresh=true

**Comment by arthurfsy2** [comment]
> @Lax 顺便说一下， keep 目前只拿了 running 的数据，如果有走路爬山自行车的数据需要改type image 比如骑行是cycling 具体对应的字段可以在这个 api 找到 https://api.gotokeep.com/config/v2/basic?refresh=true

徒步的是type=hiking，然后runinglog改成hikinglog即可

**Pull request merged**